src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationProviderRefreshEventSource.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Diagnostics.Tracing;
+
+namespace Microsoft.Extensions.Configuration.AzureAppConfiguration
+{
+    [EventSource(Name = EventSourceName)]
+    internal class AzureAppConfigurationProviderRefreshEventSource : EventSource
+    {
+        /// <summary>The name to use for the event source.</summary>
+        private const string EventSourceName = "Microsoft-Extensions-Configuration-AzureAppConfiguration-Refresh";
+
+        private const int LogDebugEvent = 1;
+        private const int LogInformationEvent = 2;
+        private const int LogWarningEvent = 3;
+
+        /// <summary>
+        ///   Provides a singleton instance of the event source for callers to
+        ///   use for logging.
+        /// </summary>
+        public static AzureAppConfigurationProviderRefreshEventSource Log { get; } = new AzureAppConfigurationProviderRefreshEventSource();
+
+        [Event(LogDebugEvent, Message = "{0}", Level = EventLevel.Verbose)]
+        public void LogDebug(string message)
+        {
+            WriteEvent(1, message);
```

Comment:
```suggestion
            WriteEvent(LogDebugEvent, message);
```

Reply:
My fault I made this change and undid it on accident